ifopt_ipopt/test/ex_test_ipopt.cc

```diff
@@ -44,7 +44,8 @@ int main()
   // 2. choose solver and options
   IpoptSolver ipopt;
   ipopt.SetOption("linear_solver", "mumps");
-  ipopt.SetOption("jacobian_approximation", "exact");
+  //ipopt.SetOption("jacobian_approximation", "exact");
```

Comment:
For the test lets keep in exact, there the jacobians are defined.

Reply:
Done.